Include/internal/pycore_frame.h

```diff
@@ -115,6 +115,11 @@ _PyFrame_InitializeSpecials(
     frame->prev_instr = _PyCode_CODE(code) - 1;
     frame->yield_offset = 0;
     frame->owner = FRAME_OWNED_BY_THREAD;
+
+    PyObject **localsarray = &frame->localsplus[0];
```

Comment:
Any compiler will do loop invariant code motion, so no need to move this out of the loop.